dialogue-core/src/main/java/com/palantir/dialogue/core/AimdConcurrencyLimiter.java

```diff
@@ -148,7 +165,7 @@ public int applyAsInt(int originalLimit, int _inFlightSnapshot) {
      * permits such that another permit can be {@link #acquire acquired}.
      */
     int getLimit() {
-        return limit.get();
+        return Ints.saturatedCast(Math.round(limit.get()));
```

Comment:
slight preference to expose this number as a double and do the cast later so that datadog dashboards could show the _precise_ number? (might help with debugging if there are a bunch of requests and we see the limit go from 120.9 -> 120.7)